src/test/outmdsd/testsocket.cc

```diff
@@ -71,6 +71,9 @@ BOOST_AUTO_TEST_CASE(Test_SocketClient_Reconnect_Error)
             auto runTimeMS = static_cast<uint32_t>((endTime-startTime)/std::chrono::milliseconds(1));
 
             BOOST_CHECK_MESSAGE((runTimeMS >= timeoutMS), "Send() index=" << i);
```

Comment:
>BOOST_CHECK_MESSAGE((runTimeMS >= timeoutMS), "Send() index=" << i); [](start = 12, length = 68)

If this check fails, only the index number is printed, not the actual timeout. I would think you'd also want to know what runTimeMS was. (There's no need to print timeoutMS, since it's hard-coded.)

BOOST_CHECK_MESSAGE(runTimeMS >= timeoutMS, "Iteration " << i << " timed out under 10ms (" << runTimeMS << "ms)");

That's the right way to fix this issue.